lib/client.js

```diff
@@ -112,16 +112,16 @@ Client.prototype.send = function() {
   // Respect IRC line length limit of 512 chars
   // https://tools.ietf.org/html/rfc2812#section-2.3
   if (message.length > 512) {
-    var parts   = message.split(':');
```

Comment:
@dequis What happens to messages that don't have a ```:``` in them? ```parts``` will be -1, which then reduces ```command``` to a single character, doesn't it? Granted, it's an unlikely scenario to have a message that's longer than 512 characters but still not have a colon... still?